scipy/stats/tests/test_mstats_basic.py

```diff
@@ -964,7 +964,7 @@ def regression_test_9033(self):
     @pytest.mark.parametrize("test", ["skewtest", "kurtosistest"])
     @pytest.mark.parametrize("alternative", ["less", "greater"])
     def test_alternative(self, test, alternative):
-        x = stats.norm.rvs(loc=10, scale=2.5, size=20, random_state=123)
+        x = stats.norm.rvs(loc=10, scale=2.5, size=30, random_state=123)
```

Comment:
This is fine - as long as it wasn't necessary to make the test pass...? What was the motivation?

Reply:
Oh, yes. It wasn't done to make the test pass but because `skewtest` requires at least 20 samples. As I add nans to some samples, I had to increase the sample size.